familias/test_views.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.core.urlresolvers import reverse
+from splinter import Browser
+
+
+class TestViewsFamilias(StaticLiveServerTestCase):
+    """Integration test suite for testing the views in the app: familias.
+
+    Test the urls for 'familias' which make up the administration dashboard.
+
+    Attributes
+    ----------
+    browser : Browser
+        Driver to navigate through websites and to run integration tests.
+    """
+
+    def setUp(self):
+        """Initialize the browser and create a user, before running the tests.
+
+        """
+        self.browser = Browser('chrome')
+
+    def tearDown(self):
+        """At the end of tests, close the browser.
+
+        """
+        self.browser.quit()
```

Comment:
and obviously it includes the removal of this line too. Actually, you don't need to declare setup and teardown at all.